docs/user-guide/data-reduction.md

```diff
@@ -1,90 +1,96 @@
-# Reflectometry data reduction procedure
+# Focused reflectometry data reduction
```

Comment:
Please rename the file. We normally match the title with the filename to make it easier to find.